docs/spec/api.md

```diff
@@ -618,26 +618,6 @@ The "digest" parameter must be included with the PUT request. Please see the
 [_Completed Upload_](#completed-upload) section for details on the parameters
 and expected responses.
 
-Additionally, the upload can be completed with a single `POST` request to
```

Comment:
@stevvooe I _am_ only guessing about what you guys want to update for the currently released docs-set.

would it be possible for you guys to adopt https://github.com/docker/docker/pull/22333 and thus label docs PR's that should go public?